web/styles/zulip.css

```diff
@@ -1368,6 +1380,10 @@ div.focused-message-list {
 
     a {
         color: inherit;
+
+        strong {
+            white-space: pre;
+        }
```

Comment:
This selector seems to be too broad; we should scope it with a specific class, both for better performance and for avoiding unexpected application of the style to code that doesn't intend it.

(The `a` styling above has a similar issue, but let's try to keep the scope for this PR bounded)